WBooks/Detail View/BookCommentCell.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  BookCommentCellTableViewCell.swift
+//  WBooks
+//
+//  Created by Gaston Maspero on 04/06/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class BookCommentCell: UITableViewCell, NibLoadable {
+
+    @IBOutlet weak var profilePic: UIImageView! {
+        didSet {
+            self.profilePic.layer.borderWidth = 1.5
+            self.profilePic.layer.masksToBounds = false
+            self.profilePic.layer.borderColor = UIColor(red: 73/255, green: 194/255, blue: 1, alpha: 1).cgColor
```

Comment:
Create a `UIColor` extension, and set all app's colors as computed properties

Reply:
ok!